src/item/views/Item.tsx

```diff
@@ -55,16 +55,16 @@ import ItemVideoDescription from '../components/ItemVideoDescription';
 import FragmentAddToCollection from '../components/modals/FragmentAddToCollection';
 import { GET_ITEM_BY_ID } from '../item.gql';
 
-import './Item.scss';
 import { getProfileName } from '../../authentication/helpers/get-profile-info';
+import './Item.scss';
 
 interface ItemProps extends RouteComponentProps {}
 
 const Item: FunctionComponent<ItemProps> = ({ history, match }) => {
 	const videoRef: RefObject<HTMLVideoElement> = createRef();
 
 	const [itemId] = useState<string | undefined>((match.params as any)['id']);
-	const [time, setTime] = useState<number>(0);
```

Comment:
dit moet nog terug enabled worden om timestamps in de url toe te staan maar dan nu met flowplayer, dus ik zou dit niet weg doen, mss een todo toe voegen op die regel?

Reply:
Added TODO